### ISSUE TITLE
{panelIndex} support on `removePanelText`

### ISSUE THREAD

**Issue by kiisodj** [surveyjs/survey-library#10044]
**Is your feature request related to a problem? Please describe.**
Accessibility and usability is compromoised with many items in a dynamic panel because the 'remove' buttons all have the same label.

**Describe the solution you'd like**
For Dynamic Panels, it would be useful to include the {panelIndex} in the 'removePanelText' button label, so that each button has a unique label when adding multiple items. 

The following example doesn't work, it would be good if the button labels rendered with dynamic panel number.

```
{
  "pages": [
    {
      "name": "page1",
      "elements": [
        {
          "type": "paneldynamic",
          "name": "question1",
          "title": "Email addresses",
          "templateElements": [
            {
              "type": "text",
              "name": "question2",
              "title": "Email address {panelIndex}",
              "inputType": "email"
            }
          ],
          "removePanelText": "Remove email address {panelIndex}"
        }
      ]
    }
  ]
}
```

**Comment by JaneSjs** [comment]
Hi @kiisodj,
If you wish to display the current panel index within the Remove Panel button's caption, implement the [`SurveyModel.onGetPanelFooterActions`](https://surveyjs.io/form-library/documentation/api-reference/survey-data-model#onGetPanelFooterActions) function and create a custom action which would display the current panel index. Consider the following demo: [View Plunker](https://plnkr.co/edit/oQqxr3Abqe2ntKPO).
```
survey.onGetPanelFooterActions.add((sender, options) => {
        options.actions.length = 0;
        const panel = options.panel;
        const dynamicPanel = panel.parentQuestion;
        const panelIndex = panel.parentQuestion.panels.indexOf(options.panel) + 1;        
        const customRemovePanelAction = {
            id: "remove-panel",
            title:`Remove email address ${panelIndex}`,
            innerCss: "sd-action--negative sd-paneldynamic__remove-btn",            
            action: () => {
                dynamicPanel.removePanel(panel);
            }
        };
        options.actions.push(customRemovePanelAction);
});
```
Please let me know if this option works for you.
